pkg/convert/testdata/states/range/import.json

```diff
@@ -1,8 +1,8 @@
 [
   {
     "Type": "simple:index:resource",
-    "Name": "a_resource-goodbye",
```

Comment:
ah this file is temperamental, it works both ways but just reset it back to master to minimize log noise.